.changeset/tidy-readers-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+'@aws-amplify/deployed-backend-client': patch
```

Comment:
```suggestion
'@aws-amplify/deployed-backend-client': minor
```

This should be minor while in `0.x` version range. (which means breaking change).

Reply:
Good catch. I mis-hit the enter...